src/components/Savings/SavingsCard/SavingsCardRow/index.tsx

```diff
@@ -86,11 +92,8 @@ export const SavingsCardRow = ({
                             size="sm"
                             noShadow={true}
                             borderRadius="6px"
-                            requireChain={'CELO'}
-                            onClick={() => {
-                                getData({ event: 'savings', action: 'startClaim' })
-                                toggleModal('claim')
-                            }}
+                            requireChain={SupportedV2Networks[requiredChain] as keyof typeof SupportedV2Networks}
+                            onClick={() => toggleSavingsModal('claim')}
```

Comment:
buttons could be also optimized, inline callbacks could be removed:

```javascript
const ModalButton = ({ chain, type, title, toggleModal, ...props }) => {
  const onClick = useCallback(() => {
         sendGa({ event: 'savings', action: 'start' + type })
         toggleModal(type)
     },
     [toggleModal, type]
   )

  return (
    <ActionOrSwitchButton
        width="130px"
        size="sm"
        borderRadius="6px"
        requireChain={chain}
        noShadow={true}
        onClick={onClick}
        {...props}
    >
        {' '}
        {title}{' '}
    </ActionOrSwitchButton>
  )
}
// ...
<ModalButton
    chain={SupportedV2Networks[requiredChain] as keyof typeof SupportedV2Networks}
    type="withdraw"
    title={i18n._(t`Withdraw G$`)}
/>
                          
```

Reply:
Added